components/usage/pkg/apiv1/billing.go

```diff
@@ -318,12 +333,7 @@ func balancesForStripeCostCenters(ctx context.Context, cm *db.CostCenterManager,
 
 func convertStripeCustomer(customer *stripe_api.Customer) *v1.StripeCustomer {
 	return &v1.StripeCustomer{
-		Id: customer.ID,
-	}
-}
-
-func convertDBStripeCustomerToResponse(cus db.StripeCustomer) *v1.StripeCustomer {
-	return &v1.StripeCustomer{
-		Id: cus.StripeCustomerID,
+		Id:       customer.ID,
+		Currency: customer.Metadata[stripe.PreferredCurrencyMetadataKey],
```

Comment:
What should happen if the Currency field doesn't exist on the metadata?

Reply:
I'm mostly asking how strong of a consistency do we want to enforce here. I'd aim for making it impossible to reach this state. Storing the currency on the DB makes this easier as well.

Reply:
That bends it too much. Fair question, indeed. 
It seems like we should not rely on stripe metadata. Maybe persisting it into Stripe customer is ok, but then communicate, that the source of truth is in Gitpod DB. 

🤔